checks/checks2.yaml

```diff
@@ -0,0 +1,57 @@
+# Copyright 2021 Security Scorecard Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This is the source of truth for all check descriptions and remediation steps.
+# Run `cd checks/main && go run /main` to generate `checks.json` and `checks.md`.
+checks:
+  Binary-Artifacts:
+    description: >-
+      This check tries to determine if a project has binary artifacts in the source repository.
+      These binaries could be compromised artifacts. Building from the source is recommended.
+    url: https://github.com/ossf/scorecard/blob/main/checks/checks.md#binary-artifacts
+    remediation:
+      - >-
+        Remove the binary artifacts from the repository. 
+    errors:
```

Comment:
these error list may seem akward (as I describe in the PR description). It's needed to support more complex checks like the pinning check that contains many sub-checks. It's easy to leave the error list empty if we want AND not call `Fail("E01)` in the code. The display of errors will automatically adjust.

Reply:
yes, thanks for the suggestion. I've used better names. PTAL.